command/ca/offline.go

```diff
@@ -228,6 +229,23 @@ func (c *offlineCA) GenerateToken(ctx *cli.Context, typ int, subject string, san
 		return "", err
 	}
 
+	switch p := p.(type) {
+	case *provisioner.OIDC: // Run step oauth
```

Comment:
just leaving a marker here for myself, hard to tell if it was addressed in a future commit.